quickstarts/apm-transaction-analysis/config.yaml

```diff
@@ -0,0 +1,25 @@
+id: 870dc0f9-57d5-4532-a985-46f9aba019b5
```

Comment:
Let's remove this line, the system will auto generate an Id for new quickstarts!
```suggestion
```

Reply:
resolved.